service/storage_service.hh

```diff
@@ -2347,8 +2347,8 @@ public:
         return bool(_mc_sstable_feature);
     }
 
-    bool cluster_supports_cdc() const {
-        return bool(_cdc_feature);
+    const gms::feature& cluster_supports_cdc() const {
```

Comment:
Can you explain why returning bool is not enough in the commit log?

Reply:
@haaawk

Reply:
not a part of this patchset anymore